include/seqan3/alignment/matrix/alignment_coordinate.hpp

```diff
@@ -346,10 +346,10 @@ class alignment_coordinate
 template <typename coordinate_type>
 //!\cond
     requires std::Same<remove_cvref_t<coordinate_type>, alignment_coordinate> ||
-             detail::is_type_specialisation_of_v<remove_cvref_t<coordinate_type>,
-                                                 detail::advanceable_alignment_coordinate>
+             detail::is_value_specialisation_of_v<remove_cvref_t<coordinate_type>,
```

Comment:
`value`_specialisation? o.O

Reply:
yeah; `detail::advanceable_alignment_coordinate` is defined as `template <enum> advanceable_alignment_coordinate;` and thus needs `is_value_specialisation_of_v`